docker/docker-compose.yml

```diff
@@ -43,24 +43,17 @@ services:
 
   # RackHD services
 
-  core:
-    build: "../on-core"
-    image: rackhd/on-core:${TAG}
-    command: "/bin/echo \"exit\""
-
-  tasks:
-    build: "../on-tasks"
-    image: rackhd/on-tasks:${TAG}
-    command: "/bin/echo \"exit\""
+  #  core:
+  #    build: "../on-core"
+  #    image: rackhd/on-core:latest
+  #    command: "/bin/echo \"exit\""
```

Comment:
The docker-compose-elk.yml is copied from the old file: docker-compose.yml. I have fix it done.